gridcomps/History/Sampler/MAPL_TrajectoryMod_smod.F90

```diff
@@ -878,13 +938,25 @@
          real(kind=REAL32), pointer :: p_acc_3d(:,:),p_acc_2d(:)
          real(kind=REAL32), pointer :: p_acc_rt_3d(:,:),p_acc_rt_2d(:)
          real(kind=REAL32), pointer :: p_src(:,:),p_dst(:,:)
+         real(kind=REAL32), pointer :: p_dst_rt(:,:)
+
+         type(ESMF_Field) :: acc_field_2d_chunk, acc_field_3d_chunk, chunk_field
+         real(kind=REAL32), pointer :: p_acc_chunk_3d(:,:),p_acc_chunk_2d(:)
 
          integer :: is, ie, nx
          integer :: lm
          integer :: rank
          integer :: status
          integer :: j, k, ig
          integer, allocatable :: ix(:)
+         type(ESMF_VM) :: vm
+         integer :: mypet, petcount, mpic, iroot
+
+         integer :: na, nb, nx_sum, nsend
+         integer, allocatable :: RecvCount(:), displs(:)
+         integer :: i, ierr
+         integer, allocatable :: nsend_v, recvcount_v(:), displs_v(:)
```

Comment:
Did you intend for `nsend_v` to be an **allocatable** **scalar**?  That's a fairly unusual design choice.  I'm guessing you meant it to be nonallocatable.